### PR TITLE
Headless chrome

### PR DESCRIPTION
**Pull request by WilcoFiers**
I had some difficulties figuring out why our tests stopped passing. I think it has something to do with PhantomJS, but it was very hard to debug. So instead I decided, since we should replace PhantomJS anyway, that time was better spent figuring out how to get headless Chrome working. So that's what this PR does:

- It add all that stuff that was in #28 
- Removes Standalone PhantomJS 
- Replaces it with Chrome, running in --headless mode

This does mean that we're no longer shipping axe-cli with its own browser. As the default, it needs CHrome 59+ to be installed in a predictable location. Axe-cli installs chromedriver though.

I am working on a few other things that I want to ship in 2.0. Mainly better test coverage and the ability to run axe-core@next. I'll have those in separate PRs.

### CODE REVIEW

**Comment by WilcoFiers** [comment]
Not sure why the tests are failing now... It worked before: https://circleci.com/gh/dequelabs/axe-cli/32 I'll figure it out.

**Comment by WilcoFiers** [comment]
Temporary fix for the issue. I need to figure out how to install a chrome driver in Circle. I'll do that as part of my test coverage work.

File: index.js [hunk 1]
```diff
@@ -19,22 +19,30 @@ program.version(version)
 .option('-b, --browser [browser-name]', 'Which browser to run (Webdriver required)')
 .option('-s, --save [filename]', 'Save the output as a JSON file. Filename is optional')
 .option('-d, --dir <path>', 'Output directory')
-.option('-a, --axe-source', 'Path to axe.js file')
+.option('-a, --axe-source <path>', 'Path to axe.js file')
```
Comment (marcysutton) [inline index.js]: This change and the conditional below seem unrelated to this PR

File: readme.MD [hunk 2]
```diff
@@ -8,7 +8,7 @@ Provides a command line interface for [aXe](https://github.com/dequelabs/axe-cor
 
 ## Getting Started
 
-Install [Node.js](https://docs.npmjs.com/getting-started/installing-node) if you haven't already. This project requires Node 6+.
+Install [Node.js](https://docs.npmjs.com/getting-started/installing-node) if you haven't already. This project requires Node 6+. By default, axe-cli runs Chrome in headless mode, which requires Chrome 59 or up.
```
Comment (marcysutton) [inline readme.MD]: This should go on a separate line since it's unrelated to Node.

File: readme.MD [hunk 3]
```diff
@@ -59,9 +59,13 @@ If you want to only test a specific area of a page, or wish to exclude some part
 
 Axe-cli will look for locally available versions of axe-core. If the directory from where you start axe-cli has an `axe.js` file, or has a `node_modules` directory with axe-core installed in it. Axe-cli will use this version of axe-core instead of the default version installed globally.
 
+To specify the exact file axe-core file axe-cli should use, you can use the `--axe-source` flag (`-a` for short), with a relative or absolute path to the file.
```
Comment (marcysutton) [inline readme.MD]: In the future we should put stuff like this into a separate PR to keep things focused.

File: test/webdriver.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+	it('creates a driver', done => {
+		startDriver(config)
+			.then((config) => {
+				assert.isObject(config.driver)
+				assert.isFunction(config.driver.manage)
+			})
+			.then(done, done)
+	})
+
+	xit('sets the config.browser as the browser', done => {
```
Comment (marcysutton) [inline test/webdriver.js]: This test is commented out, should we remove it? Or fix it

**Comment by marcysutton** [comment]
@WilcoFiers the tests fail for me locally as well, even after running a clean `npm install`. Puppeteer is out now, perhaps we should look into that instead? https://github.com/GoogleChrome/puppeteer

**Comment by marcysutton** [comment]
This works for me now, nice work.

**Comment by mohanraj-r** [comment]
@WilcoFiers Curious to know why this was not merged? Are there alt plans / WIP ?
